README.md

```diff
@@ -500,11 +499,49 @@ private int value;
 
 <a name="classes-constructors--tostring"></a><a name="9.2"></a>
 - [9.2](#classes-constructors--tostring) Add always a toString method
-for entities and models. Why: You almost always have the need to log
-the current state of a pojo. And that is when you release you get
-something like this `[Lcom.foo.Object;@28a418fc`. So keep yourself
-from this and create a `toString`-Method. Also todays IDEs can create
-it automatically.
+  for entities and models. Why: You almost always have the need to log
+  the current state of a pojo. And that is when you release you get
+  something like this `[Lcom.foo.Object;@28a418fc`. So keep yourself
+  from this and create a `toString`-Method. Also todays IDEs can create
+  it automatically.
+
+<a name="classes-constructors--order"></a><a name="9.3"></a>
+- [9.3](#classes-constructors--order) Arrange methods, variables, constructors,
+interfaces etc. in a consistent order. Why: It supports your team 
+members to understand your source code faster.
+
```

Comment:
Good point. But this is to shallow. Please propose an order, that we should use. Otherwise this is just a suggestion no one applies or even worse everyone applies but in a different manner.